config/test/test-setup.ts

```diff
@@ -9,6 +9,8 @@ import {StyleSheetTestUtils} from "aphrodite";
 import jestSerializerHtml from "jest-serializer-html";
 import {addSerializer} from "jest-specific-snapshot";
 
+import "@testing-library/jest-dom";
```

Comment:
We had sprinkled this in most test files. This makes starting a new test file just that little bit easier!